src/swap/components/SwapAmountInput.tsx

```diff
@@ -125,14 +127,17 @@ export function SwapAmountInput({
       </div>
       <div className="flex w-full items-center justify-between">
         <TextInput
-          className="w-full border-[none] bg-transparent font-display text-[2.5rem] text-gray-900 leading-none outline-none"
+          className={cn(
+            'w-full border-[none] bg-transparent font-display text-[2.5rem]',
+            'leading-none outline-none',
+            hasInsufficientBalance ? 'text-[red]' : 'text-gray-900',
```

Comment:
Use the new `color` theme object here.

`color` options are:
```
export const color = {
  inverse: 'text-inverse',
  foreground: 'text-foreground',
  foregroundMuted: 'text-foreground-muted',
  error: 'text-error',
  primary: 'text-primary',
  success: 'text-success',
  warning: 'text-warning',
  disabled: 'text-disabled',
} as const;
```

```suggestion
            hasInsufficientBalance ? color.error : color.foreground,
```